db/migrate/20230711150647_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :Title
+      t.text :Text
+      t.integer :CommentsCounter
+      t.integer :LikesCounter
+
+      t.timestamps
```

Comment:
- [ ] Please include a reference to the "users" table for the author relationship by adding `t.references :author`, `foreign_key: { to_table: 'users' }` and specify any necessary constraints or validations that might be missing for the "posts" table creation. :wink:

Reply:
I have implemented this requirement. You didn't check the other migrations. You can confirm this from the schema.rb file

Please check this migration file
https://github.com/prius2055/rails-blogging-app/blob/760daceb066491b3fed63429c1a60cecf07cd262/db/migrate/20230711163627_add_user_ref_to_posts.rb#L1

schema file

https://github.com/prius2055/rails-blogging-app/blob/760daceb066491b3fed63429c1a60cecf07cd262/db/schema.rb#L1